src/features/user/TreeMapper/Analytics/components/TreePlanted/index.tsx

```diff
@@ -114,6 +95,18 @@ export const TreePlanted = () => {
 
   const [timeFrame, setTimeFrame] = useState<TIME_FRAME | null>(null);
 
+  const { makeRequest } = useNextRequest<{
+    data: IDailyFrame[] | IWeeklyFrame[] | IMonthlyFrame[] | IYearlyFrame[];
+  }>(
+    `/api/data-explorer/trees-planted?timeFrame=${timeFrame}`,
+    HTTP_METHOD.POST,
+    {
+      projectId: project?.id,
+      startDate: fromDate,
+      endDate: toDate,
+    }
+  );
```

Comment:
Even though there isn't a scope of conflict here, adding an alias for `makeRequest` here would make the `fetchPlantedTrees` function a little more readable.

Reply:
@mohitb35 So the idea behind `useNextRequest` hook is to make a single component responsible of making a request thus, the function name `makeRequest`. Though there could be an exception where we can use it more than once which could result in conflicts so using alias there make sense but not in this case.